src/android/android-backend.cpp

```diff
@@ -38,18 +39,29 @@ namespace librealsense {
         }
 
         std::vector<uvc_device_info> android_backend::query_uvc_devices() const {
-            return usb_host::device_watcher::query_uvc_devices();
+            return device_watcher_usbhost::instance()->query_uvc_devices();
         }
 
-        std::shared_ptr<usb_device> android_backend::create_usb_device(usb_device_info info) const {
-            throw std::runtime_error("create_usb_device Not supported");
+        std::shared_ptr<command_transfer> android_backend::create_usb_device(usb_device_info info) const {
```

Comment:
the original `usb_device `was an empty class which inherited `command_transfer`, since I removed `usb_device `I needed to replace it with `command_transfer `but I don't want to make more changes to the backend API in this PR unless it is necessary .